src/Core/src/Platform/Android/ProgressBarExtensions.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Android.Content.Res;
+using Android.OS;
+using AProgressBar = Android.Widget.ProgressBar;
+
+namespace Microsoft.Maui
+{
+	public static class ProgressBar
+	{
+		public const int Maximum = 10000;
+	}
```

Comment:
nit: Could we maybe put this in the extensions class? Or maybe we should move the slider max out of that one.
I am also more concerned about potential naming clashes with libraries if they include maui and maui controls. We have things like Color and all that so it will probably be included.

Reply:
Maybe we can alo just use int.MaxValue here so that it is as smooth as possible. Although not as essential as with slider as that had real values to pull and not a percent.

Reply:
Moved directly to the `ProgressBarExtensions`.